<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1680</width>
    <height>1050</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(135, 135, 135);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1682</width>
      <height>1031</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="scoreVerticalLayout"/>
     </item>
     <item>
      <widget class="QTabWidget" name="instrumentTabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>1680</width>
         <height>900</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>10000</width>
         <height>900</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>600</width>
         <height>400</height>
        </size>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::ActionsContextMenu</enum>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;乐器界面&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string>color: rgb(255, 255, 255);</string>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="pianoTab">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>100</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <attribute name="title">
         <string>钢琴</string>
        </attribute>
        <widget class="QScrollArea" name="pianoScrollArea">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1676</width>
           <height>601</height>
          </rect>
         </property>
         <property name="widgetResizable">
          <bool>false</bool>
         </property>
         <widget class="QWidget" name="pianoScrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>4398</width>
            <height>589</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="drumkitTab">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <attribute name="title">
         <string>架子鼓</string>
        </attribute>
        <widget class="QScrollArea" name="scrollArea">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1676</width>
           <height>864</height>
          </rect>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="drumScrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1674</width>
            <height>862</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1680</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>文件(F)</string>
    </property>
    <addaction name="newAction"/>
    <addaction name="deleteAction"/>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
   </widget>
   <widget class="QMenu" name="editMenu">
    <property name="title">
     <string>编辑(E)</string>
    </property>
    <addaction name="separator"/>
    <addaction name="fullAction"/>
    <addaction name="halfAction"/>
    <addaction name="quarterAction"/>
    <addaction name="eighthAction"/>
    <addaction name="sixteenthAction"/>
    <addaction name="restAction"/>
   </widget>
   <widget class="QMenu" name="settingMenu">
    <property name="title">
     <string>乐器设置(P)</string>
    </property>
    <addaction name="pianoAction"/>
    <addaction name="volumeAction"/>
    <addaction name="metronomeAction"/>
    <addaction name="metronomeSettingAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="editMenu"/>
   <addaction name="settingMenu"/>
  </widget>
  <action name="newAction">
   <property name="text">
    <string>新的工程 (Ctrl + N)</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>保存 (Ctrl + S)</string>
   </property>
  </action>
  <action name="openAction">
   <property name="text">
    <string>打开文件 (Ctrl + O)</string>
   </property>
  </action>
  <action name="pianoAction">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>键位设置(P)</string>
   </property>
  </action>
  <action name="drumAction">
   <property name="text">
    <string>架子鼓键位设置(D)</string>
   </property>
  </action>
  <action name="metronomeAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>节拍器开启/关闭(Alt + M)</string>
   </property>
   <property name="priority">
    <enum>QAction::HighPriority</enum>
   </property>
  </action>
  <action name="volumeAction">
   <property name="text">
    <string>音量设置(Ctrl +V)</string>
   </property>
  </action>
  <action name="metronomeSettingAction">
   <property name="text">
    <string>节拍器设置(Ctrl+M)</string>
   </property>
  </action>
  <action name="deleteAction">
   <property name="text">
    <string>删除工程 (Ctrl + D)</string>
   </property>
  </action>
  <action name="fullAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>全音符 （Alt + F)</string>
   </property>
  </action>
  <action name="halfAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>半音符 (Alt + H)</string>
   </property>
  </action>
  <action name="quarterAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>四分音符 (Alt + Q)</string>
   </property>
  </action>
  <action name="eighthAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>八分音符 (Alt + E)</string>
   </property>
  </action>
  <action name="sixteenthAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>十六分音符 （Alt + S)</string>
   </property>
  </action>
  <action name="restAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>休止 (Alt + R)</string>
   </property>
  </action>
  <action name="playFromStartAction">
   <property name="text">
    <string>从头播放当前乐谱 (Alt + O)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
